ci-services/drone.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const env = process.env
+
+/**
+ * Should update the `package-lock.json`
+ */
+function shouldUpdate () {
+  let re = /^(chore|fix)\(package\): update [^ ]+ to version.*$/mi
+  return re.test(env.DRONE_COMMIT_MESSAGE)
+}
+
+module.exports = {
+  // The GitHub repo slug
+  repoSlug: env.DRONE_REPO,
+  // The name of the current branch
+  branchName: env.DRONE_COMMIT_BRANCH,
+  // Is this the first push on this branch
+  // i.e. the Greenkeeper commit
+  firstPush: shouldUpdate(),
+  // Is this a regular build
+  correctBuild: env.DRONE_BUILD_EVENT=== 'push',
```

Comment:
Drone has different events for build types in the case of what greenkeeper is up to it would be a `push` to the branch.